pyrsa/rdm/rdms.py

```diff
@@ -357,7 +357,9 @@ def sort_by(self, **kwargs):
         """Reorder the patterns by sorting a descriptor
 
         Pass keyword arguments that correspond to descriptors,
-        with value 'alpha'. 
+        with value indicating the sort type. Supported methods:
+            'alpha': sort alphabetically (using np.sort)
+            list/np.array: specify the new order explicitly
```

Comment:
also maybe clarify that the values in the list/array should correspond to the descriptor values; as opposed to just the indices that we'd pass to `reorder()`